OpenRA.Mods.Common/Traits/Multipliers/DamageMultiplier.cs

```diff
@@ -0,0 +1,34 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using OpenRA.GameRules;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Damage taken by this actor is multiplied based on upgrade level.",
+		"Decrease to increase actor's apparent strength.",
+		"Use 0 to make actor invulnerable.")]
+	public class DamageMultiplierInfo : UpgradeMultiplierTraitInfo, ITraitInfo
+	{
+		public DamageMultiplierInfo()
+			: base(new int[] { 91, 87, 83, 65 }) { }
```

Comment:
I agree with @penev92 here - there's no reason why these traits can't default to 100 (no modifier), and then let mods define their own curves.